alarm/uboot-rock64/PKGBUILD

```diff
@@ -13,20 +13,18 @@ license=('GPL')
 backup=('boot/boot.txt' 'boot/boot.scr')
 makedepends=('bc' 'git' 'rockchip-tools')
 install=${pkgname}.install
-_commit_rkbin=f64ded672836b47db3f6a0d04d8ccb4180fb3d55
-source=("ftp://ftp.denx.de/pub/u-boot/u-boot-${pkgver/rc/-rc}.tar.bz2"
+_commit_rkbin=0b8a4e66958b5d3d87fc61fc4bd8e069c79e931a
+source=("git+https://gitlab.denx.de/u-boot/u-boot.git#commit=v${pkgver}"
```

Comment:
Leave this as the tarball source.

Reply:
yes, sir :stuck_out_tongue: 

squashing this changeset:
```
diff --git a/alarm/uboot-rock64/PKGBUILD b/alarm/uboot-rock64/PKGBUILD
index 46ad52dae..39d0d4cbe 100644
--- a/alarm/uboot-rock64/PKGBUILD
+++ b/alarm/uboot-rock64/PKGBUILD
@@ -4,7 +4,7 @@
 buildarch=8
 
 pkgname=uboot-rock64
-pkgver=2019.04
+pkgver=2019.07
 pkgrel=1
 pkgdesc="U-Boot for Rock64"
 arch=('aarch64')
@@ -14,7 +14,7 @@ backup=('boot/boot.txt' 'boot/boot.scr')
 makedepends=('bc' 'git' 'rockchip-tools')
 install=${pkgname}.install
 _commit_rkbin=0b8a4e66958b5d3d87fc61fc4bd8e069c79e931a
-source=("git+https://gitlab.denx.de/u-boot/u-boot.git#commit=v${pkgver}"
+source=("ftp://ftp.denx.de/pub/u-boot/u-boot-${pkgver/rc/-rc}.tar.bz2"
         "git+https://github.com/rockchip-linux/rkbin.git#commit=$_commit_rkbin"
         'boot.txt'
         'mkscr')
@@ -24,7 +24,7 @@ md5sums=('SKIP'
          '021623a04afd29ac3f368977140cfbfd')
 
 build() {
-  cd u-boot
+  cd u-boot-${pkgver/rc/-rc}
 
   unset CLFAGS CXXFLAGS CPPFLAGS LDFLAGS
 
@@ -34,7 +34,7 @@ build() {
 }
 
 package() {
-  cd u-boot
+  cd u-boot-${pkgver/rc/-rc}
 
   mkdir -p "${pkgdir}/boot"
 

```

Reply:
.. and the checksum

Reply:
@kmihelich built and tested